lib/APPReinforcementLearning/src/App.h

```diff
@@ -141,6 +142,9 @@ class App
     /** Ensue that the mode is only sent once */
     bool m_modeSelectionSent;
 
+    /** check if the data has been sent */
+    bool m_dataSent;
```

Comment:
Why still using this member instead of using a method local variable in the loop()?